src/main/java/jenkins/metrics/impl/JenkinsMetricProviderImpl.java

```diff
@@ -1018,7 +1018,7 @@ private void checkEnterQueue(Queue.Item i) {
                         null,
                         null);
                 executorService.submit(() -> {
-                    QueueItemMetricsListener.notifyStarted(m);
+                     QueueItemMetricsListener.notifyQueued(m);
```

Comment:
Looks like you accidentally added a 5th space:

```suggestion
                    QueueItemMetricsListener.notifyQueued(m);
```